### ISSUE TITLE
Static Legend: Can't see the end of long names

### ISSUE THREAD

**Issue by hoorayimhelping** [influxdata/giraffe#719]
Series names that are long have `...` appended to them, but there's no way to see the full name:

![image](https://user-images.githubusercontent.com/146112/147678537-1fe10947-e468-4106-a439-31c6a9800801.png)

**Comment by danatinflux** [comment]
Also on tooltip view, if that helps:

![image](https://user-images.githubusercontent.com/44617130/147689518-9a9865d8-f18c-4fe0-98dd-c553c1183c17.png)


Dashboard is here:

https://us-east-1-1.aws.cloud2.influxdata.com/orgs/27538b2ae557c94b/dashboards/07d57459735b2000?lower=now%28%29%20-%201h

**Comment by TCL735** [comment]
This is very easy to fix from a technical perspective.

However, from a design perspective, a couple of points are worth mentioning. Allowing the entire string for long names works fine for the Static Legend, as it is a fixed area in the UI with scrollbars that a user can slide. The hover tooltip, however, does not allow scrolling (since it only appears on hover and is not in a fixed location). Without scrolling, the hover tooltip would display the entire string for long names, which may lead to some columns (the first columns, from the left) taking most or even all of the available width in the viewport for the tooltip causing the later columns to be completely hidden. Since there is no way to scroll a hover tooltip, this is only a problem for the hover tooltip.

There may be a way to mitigate this even for the hover tooltip, which is to adjust the orientation (horizontal/vertical), but this mitigation may not work if there are many columns with long strings. So, there are cases where the hover tooltip cannot handle many columns with long strings. A fixed width for each column (which is the cause of this issue) is a potential mitigation for that. But, this mitigation is probably very limited in effectiveness.

In conclusion, I agree that the fixed width should be removed. But I wanted to bring up the design implications of making such a change.
